cityofaustin/janis
# 3967 - Styling/localization for privacy fallback message

# Description
adds styling/localization to the form failed to load messaging
https://github.com/cityofaustin/techstack/issues/3967








# Types of changes



# How can this be tested?



 
     
1. Go here: https://janis-3967-form-fallback-styles.netlify.com/en/police-oversight/apply-to-the-community-police-review-commission
2. Make sure privacy badger (or whatever content blocker you'd like to use) is blocking formstack
3. See styled/localized fallback message

# Checklist:
